src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignAssembledContigsSpark.java

```diff
@@ -32,7 +30,7 @@
      * memory once per partition.
      */
     public static final int NUM_ASSEMBLIES_PER_PARTITION = 400;
-    public static final int EXPECTED_BREAKPOINTS_PER_ASSEMBLY = 10;
+    public static final int EXPECTED_CONTIGS_PER_ASSEMBLY = 15;
 
     @Argument(doc = "file for assembled breakpoint output", shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME, optional = false)
```

Comment:
Should the doc be changed to "file for assembled putative breakpoint output". 
And as a general consideration, should we change form "breakpoint id" to "assembly id", to avoid confusion with the new class AssembledBreakpoint? (I probably started using this "breakpoint id" term and am the person to blame for this.)